bgpd/rfapi/rfapi_import.c

```diff
@@ -4234,32 +4234,31 @@ static void rfapiBgpTableFilteredImport(struct bgp *bgp,
 	for (rn1 = bgp_table_top(bgp->rib[afi][safi]); rn1;
 	     rn1 = bgp_route_next(rn1)) {
 
-		if (rn1->info) {
-			for (rn2 = bgp_table_top(rn1->info); rn2;
-			     rn2 = bgp_route_next(rn2)) {
+		if (!bgp_node_has_info_data(rn1))
```

Comment:
Everywhere we are getting the info/table and then checking whether it is NULL or not and based on that continue. Why different check here ? This is the only place this function is used.

Reply:
fixed